src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub mod bitvec;
```

Comment:
Are we sure we want all mods public? ATM, only `cfg`, `disassemble` and `engine` are in use externaly.